build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -30,6 +30,8 @@ public class CompileTargets
             { "osx.10.11-x64", "osx.10.10-x64" },
             { "ubuntu.14.04-x64", "ubuntu.14.04-x64" },
             { "ubuntu.16.04-x64", "ubuntu.16.04-x64" },
+            { "ubuntu.14.04-arm", "ubuntu.14.04-arm" },
+            { "ubuntu.16.04-arm", "ubuntu.16.04-arm" },
             { "ubuntu.16.10-x64", "ubuntu.16.10-x64" },
```

Comment:
Thank you! I made a mistake. It should be ubuntu.14.04-arm instead of ubuntu.16.10-arm.